pkg/cluster/minio.go

```diff
@@ -0,0 +1,205 @@
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/replicatedhq/ekco/pkg/util"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func (c *Controller) ScaleMinioStatefulset(ctx context.Context) error {
+	currentMinioSS, err := c.Config.Client.AppsV1().StatefulSets("minio").Get(ctx, "ha-minio", metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("get ha-minio statefulset: %w", err)
+	}
+
+	scaleString := currentMinioSS.Annotations["kurl.sh/desired-scale"]
```

Comment:
I'd prefer to put it in kurl, since it's tied to the command that the containers run
Also it can't ever be changed for an existing statefulset, so...